spec/classes/mod/disk_cache_spec.rb

```diff
@@ -58,40 +41,19 @@
     end
   end
 
-  context 'on a RedHat 6-based OS' do
+  context 'on a RedHat 7-based OS' do
     let :facts do
       {
         id: 'root',
         kernel: 'Linux',
         osfamily: 'RedHat',
         operatingsystem: 'RedHat',
-        operatingsystemrelease: '6',
+        operatingsystemrelease: '7',
```

Comment:
Do you also need:  `concat_basedir: '/dne',` here?

Reply:
After chatting all of these should be removed. 
A rebase has slipped them back in.

Reply:
That should be all them removed hopefully.